frontend/src/pages/user_home.tsx

```diff
@@ -35,75 +36,87 @@ export function UserHomePage() {
-                  'bg-white dark:bg-neutral-900',
-                  'font-medium',
-                  'rounded-lg',
-                  'border',
-                  'border-gray-200 dark:border-neutral-600',
-                  'hover:shadow-lg',
-                  'hover:scale-105',
-                  'transition-all',
-                  'break-word',
-                )}
-              >
-                <IconButton
-                  label={'Delete Document'}
-                  icon={IoMdTrash}
-                  className={clsx('self-end -m-2')}
-                  size={28}
-                  onClick={(e) => {
-                    e.preventDefault();
-                    // TODO: Replace with modal
-                    if (confirm(`Are you sure you want to delete ${doc.name}?`)) {
-                      // mutate marks the document list as stale, so SWR refreshes it
-                      deleteDocument({ document_id: doc.id }).then(() => mutate());
-                    }
-                  }}
-                />
-                {doc.name}
-              </Link>
-            </li>
-          );
-        })}
+        {data?.map((doc) => (
+          <DocumentCard key={doc.id} doc={doc} mutate={mutate} />
+        ))}
 
-        <li>
-          <Link
-            title="create new document"
-            aria-label="create new document"
-            to={`/new`}
-            className={clsx(
-              'block',
-              'p-4',
-              'aspect-square',
-              'bg-white dark:bg-neutral-900',
-              'font-medium',
-              'rounded-lg',
-              'border',
-              'border-gray-200 dark:border-neutral-600',
-              'hover:shadow-lg',
-              'hover:scale-105',
-              'transition-all',
-              'flex',
-            )}
-          >
-            {' '}
-            <AiOutlinePlus size={28} />
-          </Link>
-        </li>
+        <NewDocumentCard />
       </ul>
 
       <Version />
     </AppContainer>
   );
 }
+
+function DocumentCard({
+  doc,
+  mutate,
+}: {
+  doc: {
+    changed_at: string;
+    created_at: string;
+    id: string;
+    media_files: { content_type: string; tags: string[]; url: string }[];
+    name: string;
+  };
+  mutate: () => void;
+}): JSX.Element {
+  return (
+    <LinkCard to={`document/${doc.id}`}>
+      <IconButton
+        label={'Delete Document'}
+        icon={IoMdTrash}
+        className={clsx('self-end -m-2')}
+        size={28}
+        onClick={(e) => {
+          e.preventDefault();
+          // TODO: Replace with modal
+          if (confirm(`Are you sure you want to delete ${doc.name}?`)) {
+            // mutate marks the document list as stale, so SWR refreshes it
+            deleteDocument({ document_id: doc.id }).then(() => mutate());
+          }
+        }}
+      />
+      {doc.name}
+    </LinkCard>
+  );
+}
+
+function NewDocumentCard() {
+  return (
+    <LinkCard title="create new document" to={`/new`}>
```

Comment:
different capitalisation?

Reply:
what do you mean?

Reply:
All the other texts we are using are using Title Case

Reply:
Ah okay. I just kept them as they were. Fixed now